src/Models/Thread.php

```diff
@@ -121,6 +121,30 @@ public static function getAllLatest()
         return static::latest('updated_at');
     }
 
+    /**
+     * Thread relationship.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     *
+     * @codeCoverageIgnore
+     */
+    public function getPublic()
```

Comment:
What is this method supposed to do? You have two returns without a condition, so one is unreachable.
The queries aren't able to run in production: it's a polymorphic relationship but you don't use the morph relation methods, hardcoded ID of 1 and it's not static but looks like one!?

Reply:
ah yes I missed that cleanup, i'll fix